### ISSUE TITLE
Wrong error message when trying to send all your tokens

### ISSUE THREAD

**Issue by bowenwang1996** [near/near-wallet#583]
**Problem**
If I try to send all my tokens to some other account, it will silently fail and in the console it shows
```
Error: {}
    at Object.u [as parseRpcError] (rpc_errors.js:25)
    at i.sendJsonRpc (json-rpc-provider.js:60)
    at async f.signAndSendTransaction (account.js:75)
```

**Expected Behavior**
Report the real error to users (`LackBalance`).

**Steps to reproduce**

1. Go to wallet.betanet.near.org and register an account.
2. Try sending all the tokens to some other existing account.

**Comment by marcinbodnar** [comment]
@vgrichina @kcole16 First of all, why sending all fail?

**Comment by kcole16** [comment]
This actually should be fixed by this: #481 since we'll prevent user from sending their entire balance.

Basically, this error is due to user trying to remove their state stake (minimum balance) without deleting their account, which is a requirement to do so.

Also, I can't find the locked balance PR anymore. Did we remove it?

**Comment by marcinbodnar** [comment]
> Also, I can't find the locked balance PR anymore. Did we remove it?

It's here: #534

**Comment by kcole16** [comment]
Right, realized that after posting my message!

**Comment by marcinbodnar** [comment]
> Basically, this error is due to user trying to remove their state stake (minimum balance) without deleting their account, which is a requirement to do so.

But it's already handled with #534 changes. Right?

**Comment by marcinbodnar** [comment]
@kcole16 @vgrichina Additionally, locally and on testnet we receive `Timeout error` so I'm not sure how can we handle it?

**Comment by kcole16** [comment]
Closed by #534
